website/static/js/fangorn.js

```diff
@@ -207,10 +207,10 @@ function _fangornUploadProgress(treebeard, file, progress) {
     } else {
         column = 1;
     }
-    msgText  += 'Uploaded ' + Math.floor(progress) + '%'
+    msgText  += 'Uploaded ' + Math.floor(progress) + '%';
 
     if (progress < 100) {
-        item.notify.update(msgText, 'success', column, 0);
+        item.notify.update(msgText, 'success', column, 1);
```

Comment:
But I would like to make a distinction between a default timeout (i.e. user did not set it) vs. user setting timeout to never happen. 
If the timeout variable passed is null then it should not set timeout, if it's undefined it should set the default time. 

What is the best practice for this? Should I do this:

```
if(timeout === undefined ) {
   timeout = this.timeout
}
```